pkg/bootstrap/config.go

```diff
@@ -484,6 +484,7 @@ func extractAttributesMetadata(envVars []string, plat platform.Environment, meta
 		case "ISTIO_METAJSON_LABELS":
 			m := jsonStringToMap(val)
 			if len(m) > 0 {
+				meta.IstioMetaLabels = m
```

Comment:
Its not clear to me why the client needs to _send_ both labels. I get why the control plane needs both, but the NodeMetadata here seems inappropriate. I would expect one label field in NodeMetadata that the client sends, and then one Label field not set by client -- probably in model.proxy top level -- with the computed state.

Reply:
This may also be a breaking change since older clients do not send this.